test/sqllogictest/transform/join_fusion.slt

```diff
@@ -120,21 +120,20 @@ Explained Query:
     Filter (#5 != #11) AND (#14 >= #10) // { arity: 15 }
       Join on=(#0 = #8 AND date_to_timestamp(#11) = (#5 - 9 months)) type=delta // { arity: 15 }
         implementation
-          %0:lineitem » %1:orders[#0, date_to_timestamp(#3)]KK » %2:customer[×]
-          %1:orders » %0:lineitem[#0, (#5 - 9 months)]KK » %2:customer[×]
-          %2:customer » %0:lineitem[×] » %1:orders[#0, date_to_timestamp(#3)]KK
-        ArrangeBy keys=[[], [#0, (#5 - 9 months)]] // { arity: 8 }
-          ReadIndex on=lineitem pk_lineitem_orderkey_linenumber=[*** full scan ***] // { arity: 8 }
-        ArrangeBy keys=[[#0, date_to_timestamp(#3)]] // { arity: 4 }
-          Filter (#0) IS NOT NULL // { arity: 4 }
-            ReadIndex on=orders pk_orders_orderkey=[*** full scan ***] // { arity: 4 }
+          %0:lineitem » %1:orders[#0]KA » %2:customer[×]
+          %1:orders » %0:lineitem[#0]KA » %2:customer[×]
+          %2:customer » %0:lineitem[×] » %1:orders[#0]KA
+        ArrangeBy keys=[[], [#0]] // { arity: 8 }
+          ReadIndex on=lineitem fk_lineitem_orderkey=[*** full scan ***, delta join lookup] // { arity: 8 }
+        ArrangeBy keys=[[#0]] // { arity: 4 }
+          ReadIndex on=orders pk_orders_orderkey=[delta join lookup] // { arity: 4 }
```

Comment:
Probably a good change because of 2 less arrangements. But it's hard to tell, because all these cross joins make this plan very sensitive to the join order, depending on data characteristics.